explainer.md

```diff
@@ -119,131 +119,129 @@ function BeginVRSession(isExclusive) {
       });
 }
 ```
+Once the session is started some setup must be done to prepare for rendering. The content to present to the device is defined by a [`VRLayer`](https://w3c.github.io/webvr/#interface-vrlayer). In the initial version of the spec only one layer type, `VRWebGLLayer`, is defined and only one layer can be used at a time. This is set via the `VRSession.baseLayer` attribute. (`baseLayer` because future versions of the spec will likely enable multiple layers, at which point this would act like the `firstChild` attribute of a DOM element.) 
 
-Once the session is started some setup must be done to prepare for rendering. The content to present to the device is defined by a [`VRLayer`](https://w3c.github.io/webvr/#interface-vrlayer). In the initial version of the spec only one layer type, `VRCanvasLayer`, is defined and only one layer can be used at a time. This is set via the `VRSession.baseLayer` attribute. (`baseLayer` because future versions of the spec will likely enable multiple layers, at which point this would act like the `firstChild` attribute of a DOM element.)
+Developers may optionally take advantage of the [WEBGL_multiview extension](https://www.khronos.org/registry/webgl/extensions/proposals/WEBGL_multiview/) to both WebGL 1.0 and WebGL 2.0 for optimized multiview rendering. If the UA is unable to support the multiview option, (e.g. the supplied context does not support this extension), the `VRWebGLLayer` constructor will throw an exception to ensure developers do not try to use incompatible shaders. Additionally, developers can query the `VRWebGLLayer.multiview` attribute to determine the framebuffer's configuration.
```

Comment:
I'm not sure I'm onboard with the idea of the constructor throwing because of an option like this. We wouldn't, for example, expect `antialias: true` to throw, even though it's similarly not supported everywhere. In reality any apps that do choose to make use of the multiview capabilities will still have a fallback path (as is demonstrated in the code example below), so allowing `multiview: true` to act as a hint which can be ignored or overridden when necessary seems like a more productive path.